src/main/resources/icons/plugin-icon.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you duplicate this icon to `src/main/resources/icons/io.kestra.plugin.confluence.pages.svg`?